### DISCUSSION TITLE
PatchGenerator: overlapPercent doubled

### DISCUSSION THREAD

**Discussion by mhoibo** [FAST-Imaging/FAST#183] [General]
Hi, I have seen that the overlap percent I feed into the PatchGenerator does not correspond to the patches I get when I view them. When I put an overlap of 0.25, it seems to give me a 50 % overlap, and if I feed it an overlap of 0.125 I get an overlap of 0.25. Does anyone know what the error might be?

**Comment by smistad** [comment]
The 25% overlap is applied to all sides. Thus the total amount of overlap for a patch is 50%.
Maybe this is not the most intuitive..

**Reply by mhoibo** [reply]
okay, thank you!
